libs/coin-framework/src/api/types.ts

```diff
@@ -11,6 +11,7 @@ export type Operation = {
   value: bigint;
   fee: bigint;
   blockHeight: number;
+  block?: BlockInfo;
```

Comment:
I wonder if it is not better to remove `blockHeight` and add: 
```js
block: BlockInfo
```
With an update of `BlockInfo`:
```js
export type BlockInfo = {
  height: number;
  hash?: string;
  time?: Date;
}
```

Reply:
I thought about that too, but I don't know who use the coin-framework, so I decided to add an optional type to avoid changements for the actual users. And for the moment we need this changes only for `xrp`, `tezos` and `stellar`bin the lama adapter. If it is not more optional, I will need to update polkadot too. If it is necessary, I can make the changes

Reply:
The `coin-framework/api` module is only used by Alpaca and the coin-module adapt for it. So you can safely update this :)